cl/users/views.py

```diff
@@ -538,6 +523,8 @@ def register(request: HttpRequest) -> HttpResponse:
                     urlencode(redirect_to),
                     urlencode(user.email),
                 )
+                if not settings.DEVELOPMENT:
+                    create_neon_account.delay(user.pk)
```

Comment:
It occurs to me that when somebody registers for an account, we can't trust their email address. We can only trust it after it has been confirmed. We want to prevent information leakage. 

For example, imagine:

1. Bob makes a donation in Neon and is assigned user number 4242.
2. Malory wants to see if Bob is a donor to FLP.
3. Malory creates an account in CL using Bob's email address.
4. Malory goes to /profile/my-donations/ and sees a link to "My Neon Portal", with an `href` of "neon.com/user/?id=4242". If Bob had not donated, the `href` would have been "neon.com/login/".
5. Now Malory knows, because of the ID in the link, that Bob is a donor.

So, simple solution, don't do this until after somebody has confirmed their email address, and make sure that we handle it properly when they change (AND confirm) their email address too.